src/domain/community/user/pages/UserAccountPage.tsx

```diff
@@ -47,7 +48,7 @@ export const UserAccountPage: FC<UserAccountPageProps> = () => {
 
   const { spaceIds, virtualContributors, innovationPacks, innovationHubs } = useMemo(
```

Comment:
Not related directly to the review but maybe declare `const accounts = data?.user?.accounts;` and then use it in the memo instead of repeating `data?.user?.accounts.`